controller/pasteBinController.js

```diff
@@ -0,0 +1,28 @@
+const PasteBin = require('../models/pasteBin');
+const shortid = require('shortid');
```

Comment:
![image](https://user-images.githubusercontent.com/54318933/184859609-c4e893ce-f2ae-4f11-ba64-776e53f24db8.png)

Reply:
I will read on it and implement it